app/views/MessagesView/index.tsx

```diff
@@ -306,12 +336,13 @@ class MessagesView extends React.Component {
 	}
 }
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state: any) => ({
 	baseUrl: state.server.server,
 	user: getUserSelector(state),
 	customEmojis: state.customEmojis,
 	useRealName: state.settings.UI_Use_Real_Name,
 	isMasterDetail: state.app.isMasterDetail
 });
 
+// @ts-ignore
```

Comment:
Why?

Reply:
When I started the typescript migration, I didn't have all the knowledge about the code base.
So the `app/containers/ActionSheet/Provider.tsx` file only received `Functional Components` but in fact we use it with class components too. 

I was not planning do this change in this PR, to do it separate in the future, but I fixed. 
Thanks.